src/libs/actions/PersonalDetails.ts

```diff
@@ -49,7 +49,7 @@ Onyx.connect({
     callback: (val) => (privatePersonalDetails = val),
 });
 
-function updatePronouns(pronouns: string) {
+function updatePronouns(pronouns: string, navigate = true) {
```

Comment:
I'd much rather have the navigation happen outside of this method. Is that possible without too much work? I don't like it when the action files do the redirecting because it makes them much more difficult to reuse like you are having to deal with here. It should be handled in the calling code.